guacamole/src/main/webapp/translations/nl.json

```diff
@@ -126,8 +129,15 @@
 
     },
 
+    "DATA_SOURCE_DEFAULT" : {
+        "NAME" : "Default (XML)"
```

Comment:
"Standaard" is also correct. Computer terminology in Dutch has a lot of English in it and "default" is widely understood as well. Some Dutch computer terminology just makes my toes curl and  "Standaard" is one of them. Dutch does not have a one-word translation for default and the Dutch word for standard is often used instead. I will  change my translation to "Standaard", no problem.